src/color.rs

```diff
@@ -340,6 +344,8 @@ mod elem {
                 exec: Color::Red,
                 exec_sticky: Color::Magenta,
                 no_access: Color::AnsiValue(245), // Grey
+                acl: Color::DarkCyan,
+                security_label: Color::White,
```

Comment:
white might break for people using light colorschemes. We could do cyan I guess. Suggestions welcome.